Documentation/building/linux-instructions.md

```diff
@@ -36,7 +36,7 @@ ellismg@linux:~$ sudo apt-get update
 
 Then install the packages you need:
 
-`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext`
+`ellismg@linux:~$ sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev libunwind8 libunwind8-dev gettext liblttng-ust-dev`
```

Comment:
Hopefully, we will be able to make lttng a soft-dependency in which case, it won't be required for tracing, and this line can be reverted.